Instructions/Labs/13-implement-transitions-in-tsql.md

```diff
@@ -4,17 +4,17 @@ lab:
     module: 'Additional exercises'
 ---
 
-In this lab, you'll use T-SQL statements to see the impact of using transactions in the **AdventureWorks** database. For your reference, the following diagram shows the tables in the database (you may need to resize the pane to see them clearly).
+In this lab, you'll use T-SQL statements to see the impact of using transactions in the **AdventureWorks** database. For your reference, the following diagram shows the tables in the database (resize the pane if it can't be see clearly).

```

Comment:
Please revert to the text as it was. (1.) it's common across all of the modules in this set, and (2.) the updated text is grammatically incorrect ("see instead of seen"), and (3.) even if it was correct, it's in passive voice - we prefer active voice.